data/data/fcos-stream.json

```diff
@@ -0,0 +1,294 @@
+{
+    "stream": "testing",
+    "metadata": {
+        "last-modified": "2021-06-01T22:22:20Z"
+    },
+    "architectures": {
+        "x86_64": {
```

Comment:
Do we have support for any of the other architectures in OKD? Is the limitation availability of FCOS for the architecture? What is the user experience when selecting an architecture that is not supported in OKD? Does the user get an early, meaningful error?

Reply:
>Do we have support for any of the other architectures in OKD?

Not yet, but its planned.

> Is the limitation availability of FCOS for the architecture?

Yes, its one of the limitations. The other ones are missing release-controllers for these arches and, as a result, no non-amd64 payloads.

> What is the user experience when selecting an architecture that is not supported in OKD? Does the user get an early, meaningful error?

They'd get `<arch>: No <platform> build found` as the artifact for this architecture/platform combination not found.